docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
       {
         name: 'disabled',
         type: 'boolean',
+        description: `Used to disable the entire SelectList. See the [states](#States) variant to learn more.`,
         defaultValue: 'false',
       },
       {
         name: 'errorMessage',
         type: 'string',
-        href: 'exampleWithError',
+        description: `Used to communicate error information to the user. Be sure to localize the text. See the [states](#States) variant to learn more.`,
       },
       {
         name: 'helperText',
         type: 'string',
-        description: 'More information about how to complete the form field',
-        href: 'helperText',
+        description: `Used to provide more information about the form field. Be sure to localize the text. See the [helper text](#Helper-text) variant to learn more.`,
       },
       {
         name: 'id',
         type: 'string',
+        description:
+          'A unique identifier to connect the underlying `<select>` with the associated label',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'label',
         type: 'string',
+        description: 'The label shown above the input. Be sure to localize the label.',
       },
       {
         name: 'name',
         type: 'string',
-        href: 'basicExample',
+        description: 'Used to specify the name of the control.',
       },
       {
         name: 'onChange',
-        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        type: '({| event: SyntheticInputEvent<>, value: string |}) => void',
+        description: 'Callback triggered when the user makes any changes',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'options',
-        type: 'Array<{ label: string, value: string, disabled?: boolean }>',
+        type: 'Array<{| label: string, value: string, disabled?: boolean |}>',
+        description:
+          'The options displayed in the dropdown list. Note that ``disabled`` here is used to disable a single option. Be sure to localize the label.',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'placeholder',
         type: 'string',
-        href: 'basicExample',
+        description:
+          'If not provided, the first item in the list will be shown. Be sure to localize the text.',
       },
       {
         name: 'size',
         type: '"md" | "lg"',
         required: false,
-        description: 'md: 40px, lg: 48px',
+        description: `md: 40px, lg: 48px. See the [size](#Size) variant to learn more.`,
         defaultValue: 'md',
       },
       {
         name: 'value',
         type: 'string',
-        description: 'Value that is selected.',
-        href: 'basicExample',
+        description: 'The currently-selected value',
```

Comment:
value is a _prepopulated_ value and it requires state to make the form component controlled. 

passing just value makes the options not selectable, need to use onChange to change the value's value. See error state example to see the case.

i'd be worth an example of the implementation.

Reply:
Totally makes sense. Do you have a go-to example for this? I don't necessarily think I should be writing code for these kinds of examples.

Reply:
the placeholder variant covers this

Reply:
OK, cool. I set up a Placeholder variant with onChange hooked up. I'm assuming I should migrate all the examples over to use `onChange` correct?

Reply:
We don't need to — we do offer the possibility that SelectList can be used within an uncontrolled form, basically something like
```js
<form onSubmit={handleSubmit}>
  <SelectList />
</form>
```
I'm personally not a fan of uncontrolled forms, but some people use them. We could possibly take a stronger opinion there in the future.

Reply:
👍